op-proposer/cmd/main.go

```diff
@@ -38,7 +41,8 @@ func main() {
 		},
 	}
 
-	err := app.Run(os.Args)
```

Comment:
Somehow this app was missing an interrupt waiter

Reply:
It did, although the lifecycle helper thing attached a default one. But yes, it's good to set it up in the `main`, so the signal handling is up as early as possible.